server/src/Database.hs

```diff
@@ -162,16 +172,16 @@ newtype AuthToken = AuthToken {unAuthToken :: BS.ByteString}
 instance Uniform AuthToken where
   uniformM g = AuthToken <$> uniformByteStringM 16 g
 
-getAuthTokenUser :: Transaction -> AuthToken -> IO (Maybe M.UserHandle)
+getAuthTokenUser :: Transaction -> AuthToken -> IO (Maybe M.UserAccountName)
 getAuthTokenUser (Transaction conn) (AuthToken token) = do
   result <-
     Sqlite.query
       conn
-      "select user_id from auth_tokens where token = ?;"
+      "select username from auth_tokens join users on users.id = auth_tokens.user_id where token = ?;"
```

Comment:
We should probably also call the `username` field `user_account_name` or something to match better with the model.

Reply:
Renamed all database tables and fields more according to the model